mantidproject/mantid
# SCD algorithm for normalization and detector adjustment

**Description of work.**

Adds a new algorithm to take DEMAND autoreduced data and normalize it to a given vanadium file or workspace, convert to Q-space, and allow the detector to be adjusted through a height and distance offset relative to the current detector position. The algorithm can output an MDEventWorkspace or MDHistoWorkspace.

**To test:**
Run the unit tests to verify the detector position is moved properly (`SCDAdjustSampleNormTest`).
Load in autoreduced DEMAND data and verify normalization and conversion to Q-space is being done correctly.


*There is no associated issue.*



The version of this into `ornl-next` is #29878.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
